packages/spacecat-shared-data-access/src/models/importer/import-url.js

```diff
@@ -10,89 +10,82 @@
  * governing permissions and limitations under the License.
  */
 
+/**
+ * @typedef {import('../../').ImportUrl} ImportUrl
+ */
+
 import { hasText, isValidUrl } from '@adobe/spacecat-shared-utils';
 import { Base } from '../base.js';
-import { ImportJobStatus } from './import-job.js';
-
-export const ImportUrlStatus = {
-  PENDING: 'PENDING',
-  REDIRECT: 'REDIRECT',
-  ...ImportJobStatus,
-};
+import { ImportUrlStatus } from './import-constants.js';
 
 /**
- * Creates a new ImportUrl object
- *
- * @param {Object} importUrlData
- * @returns {ImportUrl}
+ * Creates a new ImportUrl object.
+ * @param {Object} data - The data for the ImportUrl object.
+ * @return {ImportUrl} The new ImportUrl object.
  */
-const ImportUrl = (data) => {
-  const self = Base(data);
+const ImportUrlEntry = (data) => {
```

Comment:
Same question here. What does the `Entry` suffix buy us?

Reply:
I was finding that the IDE wasn't able to resolve the ImportUrl type, if this function was called ImportUrl, it didn't know to look at the type definition.  

Being that the ImportUrlEntry is essentially a helper function that isn't exposed from this module, I just added `Entry`, where it maybe more accurate to call it `construct` function maybe?

Reply:
I see. Perhaps the `Builder` suffix would work?